markdown/types_and_values.md

```diff
@@ -0,0 +1,205 @@
+**# Part One**:
+
+**## Identifying the Type of Given Values.**
+
+**### Instructions: For each of the following Elm values, write down its most specific type.**
```

Comment:
You don't need to specify that headers should be in bold font.

They'll be in bold fonts by default.

THink in terms of html. You'll never do something like `<b><h1>Part One</h1><b>`.